addons/edi/models/edi_document.py

```diff
@@ -96,6 +96,23 @@ def _default_project_id(self):
         default=False,
     )
 
+    # Controls stages for automatic processing
+    processing_level = fields.Selection(
+        [
+            ("disabled", "Disabled"),
+            ("prepare", "Prepare"),
+            ("execute", "Execute")
+        ],
```

Comment:
The default should be set to `disabled`, I believe. The empty value is technical treated similar to `disabled` (i.e., no automatic prepare/execute).

Reply:
As clarified, default should be `executed`, code revised.